### PR TITLE
Update golangci-lint/golangci-lint-action

### PR DESCRIPTION
**Pull request by michalbiesek**

### CODE REVIEW

File: scripts/golangci-lint.yml [hunk 1]
```diff
@@ -27,6 +27,6 @@ jobs:
         run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
         if: github.repository == 'prometheus/snmp_exporter'
       - name: Lint
-        uses: golangci/golangci-lint-action@v3.4.0
```
Comment (Nexucis) [inline scripts/golangci-lint.yml:30]: Can you remove this change ? It's done via dependabot and we just upgrade it to the v3.7.0

File: scripts/golangci-lint.yml [hunk 2]
```diff
@@ -27,6 +27,6 @@ jobs:
...
+        uses: golangci/golangci-lint-action@v3.6.0
         with:
-          version: v1.53.3
+          version: v1.54.1
```
Comment (Nexucis) [inline scripts/golangci-lint.yml:32]:
```suggestion
          version: v1.54.2
```

File: scripts/golangci-lint.yml (hunk 1 above)
Comment (michalbiesek) [reply scripts/golangci-lint.yml:30]: Done - removed.

File: scripts/golangci-lint.yml (hunk 2 above)
Comment (michalbiesek) [reply scripts/golangci-lint.yml]: Done (in every place)

**Review by Nexucis — APPROVED** [review]
nice thanks :)

**Pull request merged**